src/auro-radio-group.js

```diff
@@ -414,7 +415,7 @@ export class AuroRadioGroup extends LitElement {
           </p>`
         : html`
           <p class="radioGroupElement-helpText" role="alert" aria-live="assertive" part="helpText">
-            ${this.setCustomValidity}
+            ${this.errorMessage}
```

Comment:
This should be expanded to support normal help text when no error message is present from validation.

Reply:
You're right, I took this logic from datepicker which likely needs to be updated too.